### PR TITLE
New package: OpenChemistry.Avogadro2 version 1.91.0

### LINKED ISSUE #156209: [Package Request]: Avogadro 2
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.openchemistry.org/downloads/
Publisher: [Open Chemistry project](https://www.openchemistry.org/about/)
Package Name: Avogadro 2
Description: 

```
Avogadro 2In order to tackle molecular simulation and visualization challenges in key areas of materials science, chemistry and biology it is necessary to move beyond fixed software applications. The Avogadro project is in the final stages of an ambitious rewrite of its core data structures, algorithms and visualization capabilities. The project began as a grass roots effort to address deficiencies observed by many of the early contributors in existing commercial and open source solutions. Avogadro is now a robust, flexible solution that can tie in to and harness the power of VTK for additional analysis and visualization capabilities.
Avogadro 2 is a chemical editor and visualization application, it is also a set of reusable software libraries written in C++ using principles of modularity for maximum reuse. The development of the first generation Avogadro application and library is documented in our paper, and this remains the preferred method of citation at present. The motivation for rewriting Avogadro, along with improvements and changes made in Avogadro 2 are summarized in our Source article. We provide a set of permissively licensed, open source, cross platform software components in the Avogadro 2 libraries, along with an end-user application with full source code, and binaries.
The library features updated and improved rendering, where we built upon the abstraction provided by previous API, but implemented a simple scene graph. This makes use of features such as impostor sphere rendering, resulting in significant rendering speed improvements while improving the quality of the visualization. The core is built for scalability, looking to enable the analysis of larger chemical structures and simulations being produced by computational chemistry codes today. Emphasis has also been placed on making it even easier to extend, using simple Python scripts to add simulation input capabilities, and data input/output along with access to full-blown C++ plugin APIs where more control is required.
Avogadro is now able to make full use of the visualization capabilities of VTK, in addition to its own powerful rendering capabilities. This means that complex visualization, involving techniques such as volume rendering for point data, or streamlines for vector fields, will now become possible. The project is composed of two separate repositories, with the `avogadroapp’ repository offering a full demonstration of how to use the libraries in an end-user application. The `avogadrolibs’ repository contains all of the libraries, with the option to only build subsets. The development process uses distributed version control (Git), testing (CTest/CDash), and automated binary generation.
```

Package Version: `1.91.0`
Installer URL: https://github.com/OpenChemistry/avogadroapp/releases/download/1.91.0/Avogadro2-1.91.0-win64.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Related to #156209, but failed to solve it.

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/156213)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:

> Executable C:\Program Files\Avogadro2\bin\avogadro2.exe returned exit code: -1073740771

(Automated response - build 882.)

File: manifests/o/OpenChemistry/Avogadro2/1.91.0/OpenChemistry.Avogadro2.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: OpenChemistry.Avogadro2
+PackageVersion: 1.91.0
+PackageLocale: en-US
+Publisher: http://openchemistry.org/
+PublisherUrl: https://www.openchemistry.org/
+PackageName: Avogadro 2
```
Comment (Trenly) [inline manifests/o/OpenChemistry/Avogadro2/1.91.0/OpenChemistry.Avogadro2.locale.en-US.yaml:9]:
```suggestion
PackageName: Avogadro2
```
This should match Control Panel

**Review by Trenly — CHANGES_REQUESTED** [review]
Also worth noting that `1.99.0` is the latest version

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 
> Faulting application name: avogadro2.exe, version: 0.0.0.0, time stamp: 0x5b7d7773 
> Faulting module name: unknown, version: 0.0.0.0, time stamp: 0x00000000 Exception code: 0xc000041d Fault offset: 0x0000000000000000 
> Faulting process id: 0x12b8 
> Faulting application start time: 0x01dab61695f10148 
> Faulting application path: C:\Program Files\Avogadro2\bin\avogadro2.exe 
> Faulting module path: unknown Report Id: 956accb1-e848-4833-a329-f772e791275e 
> Faulting package full name:  
> Faulting package-relative application ID:   Fault bucket 1875445789161788192, type 4 Event Name: APPCRASH Response: Not available Cab Id: 0  Problem signature: P1: avogadro2.exe P2: 0.0.0.0 P3: 5b7d7773 P4: StackHash_e372 P5: 0.0.0.0 P6: 00000000 P7: c000041d P8: PCH_84_FROM_unknown+0x0000000000000000 P9:  P10:   Attached files: \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC303.tmp.dmp \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC352.tmp.WERInternalMetadata.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC353.tmp.xml \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC363.tmp.csv \\?\C:\ProgramData\Microsoft\Windows\WER\Temp\WERC373.tmp.txt  These files may be available here: \\?\C:\ProgramData\Microsoft\Windows\WER\ReportArchive\AppCrash_avogadro2.exe_edba6658a4290d49620c1424c999da124691_72ae8156_85359718-1927-4e7c-8a11-3fed3d09db1d  Analysis symbol:  Rechecking for solution: 0 Report Id: 956accb1-e848-4833-a329-f772e791275e Report Status: 268435456 Hashed bucket: 51f4df3fad9ab840ea06ec023a7d9320 Cab Guid: 0

(Automated response - build 883.)

**Comment by Dragon1573** [comment]
See https://github.com/microsoft/winget-pkgs/pull/156213#pullrequestreview-2095072487 for details.

**Pull request closed without merging**